components/massmint/useMassMint.ts

```diff
@@ -35,7 +42,7 @@ export const useCollectionForMint = () => {
   const { urlPrefix } = usePrefix()
   const queryPath = {
     rmrk: 'chain-rmrk',
-    ksm: 'chain-ksm',
+    ksm: 'chain-rmrk',
```

Comment:
since this is same path, maybe we can use general query instead?

Reply:
there is 1 ```collectionForMint.graphql``` in ```subsquid/general``` and another in ```subsquid/rmrk``` and they are tiny bit different

I considerd copy/paste the query in ```subsquid/rmrk```  into ```subsquid/ksm``` as well but that seems like a worse solution

Reply:
maybe we can create another `subsquid/kusama` or `subsquid/basilisk`. we can improve it later